### PR TITLE
Update comments array

### PR DESCRIPTION
**Pull request by BeardOfDan**

### CODE REVIEW

File: db/index.js [hunk 1]
```diff
@@ -140,9 +140,26 @@ const loadAll = (modelType = 'student') => {
...
+// ========================
+
+// hostType is the thing that the comment is on (ex. student, event, etc.)
+//   It is hostType instead of modelType, because it a part of the model, not the model itself
+// identifier is an object that is used to select the particular student, same as for loadParticular
+// comment is the comment object, it will be pushed onto the array of comments for the identified student
+const updateComments = (hostType = 'student', identifier, comment) => {
+  return getModelType(hostType)
+    .findOneAndUpdate(identifier, { $push: { 'comments': comment } }, { 'new': true })
+    .then((arg) => {
```
Comment (philcho) [inline db/index.js:158]: Why the name 'arg'? This is the entire updated set of comments, right?

Comment (BeardOfDan) [reply db/index.js:158]: Yeah, sometimes I just write 'arg' for the argument name. I should be more descriptive.

File: server.js [hunk 2]
```diff
@@ -172,10 +182,65 @@ app.post('/getParticularShoutout', (req, res, next) => {
...
+  // === Event ===
+  /*
+  {
+    "modelType" : "event",
+    "identifier": {
+      "_id": "5a0dd8307bd6aac9c8394636"
+    },
+    "comment": {
+      "name": "Phil",
+      "comment": "I can't do sign language correctly because I have crazy old man arthritis"
```
Comment (philcho) [inline server.js:224]: Is the arthritis 'crazy' or the old man?

Comment (BeardOfDan) [reply server.js:224]: While it may be a dangling participle, it does work pretty well for both, so I'm good with leaving it.

Comment (aqchin) [reply server.js:224]: Asking the real questions

File: db/index.js (hunk 1 above)
Comment (philcho) [reply db/index.js:158]: Or is this your go-to generic identifier for promises? Like "data"

File: server.js [hunk 3]
```diff
@@ -50,17 +52,25 @@ app.get('/profile', (req, res, next) => {
...
 
 app.get('/getAllStudents', (req, res, next) => {
   db.loadAll('student')
     .then((data) => {
-      res.end('Here is all of our students: \n' + JSON.stringify(data, undefined, 2));
+      res.end(JSON.stringify(data));
     })
 });
 
+app.get('/getStudent:id', (req, res, next) => {
```
Comment (philcho) [inline server.js:66]: I have a feeling we won't be needing this anymore. But we can leave it in for now, as we discussed.

File: db/index.js (hunk 1 above)
Comment (BeardOfDan) [reply db/index.js:158]: I mostly have shifted to 'data' as my default, but 'arg' was my original, and sometimes it slips in

**Review by philcho — APPROVED** [review]

**Pull request merged**